website/usermodlog.php

```diff
@@ -153,6 +156,12 @@
 		$entry = $row['entry'];
 		$fromindex = strpos($entry, " from: ");
 		if ($fromindex !== false) $entry = substr($entry, 0, $fromindex);
+
+		if (!$upperstaff) {
+			// Hide email addresses from lowerstaff
+			$entry = preg_replace($EMAIL_REGEX, "", $entry);
```

Comment:
```suggestion
			$entry = preg_replace($EMAIL_REGEX, "[email address hidden]", $entry);
```

Should make it easier to tell what happened and also to find bugs if they appear.